pubspec.yaml

```diff
@@ -1,26 +1,30 @@
 name: dart2js_info
-version: 0.5.9
+version: 0.5.10
+
 description: >
   Libraries and tools to process data produced when running dart2js with
   --dump-info.
 author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/dart-lang/dart2js_info/
 
 environment:
-  sdk: '>=2.0.0-dev.17.0 <2.0.0'
+  sdk: '>=2.0.0-dev.17.0 <3.0.0'
+
 dependencies:
-  args: '>=0.13.0 <2.0.0'
+  args: ^1.4.3
```

Comment:
@kevmoo - now that this package is for Dart 2 only, it should be ok to restrict the version of args in this way, right?

Reply:
It's good to not aggressively push updates on normal `dependencies` so we don't force churn on consumers of teh package.

...but this package is not used much outside the SDK, so in this case it's okay.